src/Enum/OrientationEnum.php

```diff
@@ -17,6 +17,10 @@ class OrientationEnum
 
     public static function getOrientations(): array
     {
-        return self::ORIENTATIONS;
+        return [
```

Comment:
self::ORIENTATIONS is no longer required?

Reply:
Moved translations to this array